.github/actions/coverage_collection/action.yml

```diff
@@ -6,7 +6,6 @@ runs:
     - name: Coverage collection
       run: |
         coverage combine
-        coverage xml
```

Comment:
why did we remove this but we still used it in the Cuda coverage collection

Reply:
`coverage combine` combines `.coverage.X` files into a `.coverage` file. Once we have a `.coverage` file describing all the tests, we can call `coverage xml` which converts a `.coverage` file into an XML summary.

If I leave this here then the output is the XML summary not the `.coverage` file and then it is much harder to combine the outputs from the linux tests and the cuda tests

Reply:
Oh I see thank you.